docs/Plugin-ChrootScan.md

```diff
@@ -13,10 +13,13 @@ The chroot_scan plugin is disabled by default. To enable it and to add files to
     config_opts['plugin_conf']['chroot_scan_opts'] = {
         'regexes': [ "core(\.\d+)?", "\.log$",],
         'only_failed': True,
+        'tarball': False,
     }
 
 The above logic turns on the chroot_scan plugin and adds corefiles and log files to the scan plugin. When the 'postbuild' hook is run by mock, the chroot_scan will look through the chroot for files that match the regular expressions in it's list and any matching file will be copied to the mock result directory for the config file. Again if you want this to be enabled across all configs, edit the `/etc/mock/site-defaults.cfg` file.
 
 When `only_failed` is set to False, then those files are always copied. When it is set to True (default when plugin enabled), then those files are only copied when build failed.
 
+when `tarball` is set to True, then instead of `chroot_scan` directory, `chroot_scan.tar.gz` is created with the directory archive.
```

Comment:
Nit `s/when/When/`.

Reply:
I'd probably move this a bit down the `only_failed option is available since mock-1.2.8` note.  If you want to declare when the new option is added, you can use `v5.5` (the next version of Mock).